pykafka/producer.py

```diff
@@ -250,3 +250,68 @@ def produce(self, messages):
         # only *some* messages when a leader changes. Therefore, we don't want
         # a random partition distribution changing that on the retry.
         self._produce(self._partition_messages(messages), 0)
+
+
+class AsyncProducer(Producer):
+    """
+    This class implements the asynchronous producer logic found in the
+    JVM driver. In inherits from the synchronous implementation.
+    """
+    def __init__(self, *args, **kwargs):
```

Comment:
-1 to not actually having args spelled out. makes it annoying to use if you've got an editor which is doing auto-complete like vim-jedi